src/more-infos/more-info-climate.html

```diff
@@ -208,6 +238,20 @@
     }.bind(this), 500);
   },
 
+  computeTargetTempHidden: function (stateObj) {
+    return stateObj.attributes.operation_mode.includes('Off');
+  },
+
+  computeHideTempRangeSlider: function (stateObj) {
+    return !stateObj.attributes.operation_mode.includes('Auto');
+  },
+
+  computeHideTempSlider: function (stateObj) {
+    return (
```

Comment:
It's for hiding normal temp slider when rangeslider is active, but I see it can maybe be combined with the off one.